examples/test_iam_identity_v1_examples.py

```diff
@@ -897,6 +897,39 @@ def test_delete_link_example(self):
         except ApiException as e:
             pytest.fail(str(e))
 
+    @needscredentials
+    def test_delete_link_by_parameters_example(self):
+        """
+        delete_link_by_parameters request example
+        """
+        CreateProfileLinkRequestLink = {}
+        CreateProfileLinkRequestLink['crn'] = (
+            'crn:v1:staging:public:iam-identity::a/' + account_id + '::computeresource:Fake-Compute-Resource'
+        )
+        CreateProfileLinkRequestLink['component_name'] = 'component_name'
+        CreateProfileLinkRequestLink['component_type'] = 'test_componenet_type'
```

Comment:
Just to be consistent. :)

```suggestion
        CreateProfileLinkRequestLink['component_name'] = 'test_component_name'
        CreateProfileLinkRequestLink['component_type'] = 'test_component_type'
```